src/AddTodoForm.js

```diff
@@ -1,20 +1,29 @@
-import React from 'react';
+import React, {useState} from 'react';
 
-const AddToDoForm = (props) => {
-    const handleAddTodo = (event) => {
-        event.preventDefault();
+const AddToDoForm = ({onAddTodo}) => {
 
-        const todoTitle = event.target.title.value;
-        console.log( todoTitle );
-        event.target.reset();
+    const [todoTitle, setTodoTitle] = useState('');
 
-        props.onAddTodo( todoTitle );
+    const handleTitleChange = (event) => {
+        const newTodoTitle = event.target.value;
+        setTodoTitle(newTodoTitle);
     }
 
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+
+        onAddTodo(
+        {
+            title: todoTitle,
+            id: Date.now(),
+        }
+        )
```

Comment:
Great job overall! Just small detail you need to add logic to reset the todoTitle state to an empty String.

Reply:
Will do!